### PR TITLE
Fix to use unit test macros that exist in Boost 1.53

### PR DESCRIPTION
**Pull request by rschutt**
Addresses https://github.com/gnuradio/gnuradio/issues/2685

### CODE REVIEW

File: gr-fft/lib/qa_fft_shift.cc [hunk 1]
```diff
@@ -39,12 +39,32 @@ BOOST_AUTO_TEST_CASE(t1)
     std::vector<int> y_even{ -4, -3, -2, -1, 0, 1, 2, 3 }; // expected result
 
     s.shift(x_even);
-    BOOST_TEST(x_even == y_even, boost::test_tools::per_element());
+    if(x_even.size() == y_even.size())
```
Comment (mbr0wn) [inline gr-fft/lib/qa_fft_shift.cc]: Can you please apply our clang-format?

Comment (rschutt) [reply gr-fft/lib/qa_fft_shift.cc]: Applied.

**Comment by marcusmueller** [comment]
Manually merged. Thank you!

**Comment by ajwolfram** [comment]
Any chance this can be applied to branch "maint-3.8", which is the suggested branch to install on the wiki page https://wiki.gnuradio.org/index.php/InstallingGR?

**Pull request closed without merging**